tests/playwright/src/model/pages/bootc-extension-page.ts

```diff
@@ -18,10 +18,10 @@
 
 import type { Page } from '@playwright/test';
 
-import { ExtensionPage } from './extension-page';
+import { ExtensionDetailsPage } from './extension-details-page';
```

Comment:
why do we have this class in this repository ? AFAIK it should be in its own repository

Reply:
This is here for some time already. 
Was also an option to have them in their particular repos, altough, if you want to test multiple external resources, you would need to share POM. Ideally on once place and this repo seems like logical place.

Reply:
it should not be there at all. Podman Desktop has no knowledge on 3rd party extension

you may have different model page for each version of the extension so it can't be there

Reply:
> if you want to test multiple external resources, you would need to share POM

I don't get it there ? Nothing related to external extension should be there, only the things concerning the pages provided by Podman Desktop

Reply:
ok, removing.

Reply:
Well this is a change from the previous agreement as I understood it, previously it was discussed that extension tests should be in the extension repos, but that the framework part could be here so that we can use it in other tests also.

For example if we move the framework part in their respective extension repos it will be much harder to create an e2e test that could test other features that might also imply some functionality from an extension, like for example if you wanted an e2e test that would use functionality from AI Lab and bootc in one test.